src/style.css

```
@@ -1 +1,102 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.searchbar {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.searchbar:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.searchbar:focus-within {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.searchbar__conainer {
+  width: 100%;
+  position: relative;
+}
+
+.searchbar__large {
+  height: 70px;
+}
+
+.searchbar__small {
+  height: 42px;
+}
+
+.searchbar__input {
+  width: 100%;
+  height: 100%;
+  padding-left: 62px;
+  box-sizing: border-box;
+  outline: none;
+
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
```

Move font-family: Avenir, sans-serif; to body and here only inherit it
```suggestion
  font-family: inherit;
```